modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -17,9 +17,9 @@ GA:: General Availability
 
 | {pipelines-title} Version 7+| Component Version | OpenShift Version | Support Status
 
-| Operator | Pipelines | Triggers | CLI | Catalog | Chains | Hub | PAC | |
+| Operator | Pipelines | Triggers | CLI | Catalog | Chains | Hub | {pac} | |
```

Comment:
why `{}` :thinking: 
is it to indicate TP ?

Reply:
Thanks for asking. Items like this between curly braces are asciidoc attributes: they import a value that's specified in a separate file. We do this to avoid naming incosistencies.